app/assets/javascripts/application.js

```diff
@@ -1,42 +0,0 @@
-// This is a manifest file that'll be compiled into application.js, which will include all the files
-// listed below.
-//
-// Any JavaScript/Coffee file within this directory, lib/assets/javascripts, vendor/assets/javascripts,
-// or vendor/assets/javascripts of plugins, if any, can be referenced here using a relative path.
-//
-// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
-// the compiled file.
-//
-// WARNING: THE FIRST BLANK LINE MARKS THE END OF WHAT'S TO BE PROCESSED, ANY BLANK LINE SHOULD
-// GO AFTER THE REQUIRES BELOW.
-//
-//= require jquery
-//= require jquery_ujs
-//= require external_scripts/bootstrap-transition.js
```

Comment:
Worth checking for each of these external scripts if they are still required.

Reply:
Mmm, I looked. Some of them are referenced in revised, and they have external script files in `external_scripts`, but otherwise none of them are referenced elsewhere. Should I delete the external script files?